project.json

```diff
@@ -12,20 +12,14 @@
     },
     "code-pushup": {
       "command": "npx dist/packages/cli",
-      "dependsOn": [
-        {
-          "projects": [
-            "cli",
-            "plugin-eslint",
-            "plugin-coverage",
-            "plugin-js-packages",
-            "plugin-lighthouse",
-            "examples-plugins",
-            "react-todos-app"
-          ],
+      "options": {
+        "persist": {
+          "format": ["json"],
+          "filename": "code-pushup",
+          "outputDir": ".code-pushup",
```

Comment:
I realize this is currently ignored because it doesn't run the executor, but why exclude `md` format and use `code-pushup` instead of the usual `report` as the filename? Please don't override the default values unless there's a good reason.

Reply:
was a debugging left over and i should have done a proper rescreening!